src/plugins/jsx/fromCodePoint.js

```diff
@@ -40,18 +40,20 @@ if (!fromCodePoint) {
       if (
         !isFinite(codePoint) || // `NaN`, `+Infinity`, or `-Infinity`
         codePoint < 0 || // not a valid Unicode code point
-        codePoint > 0x10FFFF || // not a valid Unicode code point
+        codePoint > 0x10ffff || // not a valid Unicode code point
         floor(codePoint) != codePoint // not an integer
       ) {
         throw RangeError("Invalid code point: " + codePoint);
       }
-      if (codePoint <= 0xFFFF) { // BMP code point
+      if (codePoint <= 0xffff) {
+        // BMP code point
         codeUnits.push(codePoint);
-      } else { // Astral code point; split in surrogate halves
+      } else {
+        // Astral code point; split in surrogate halves
         // https://mathiasbynens.be/notes/javascript-encoding#surrogate-formulae
         codePoint -= 0x10000;
-        highSurrogate = (codePoint >> 10) + 0xD800;
```

Comment:
Yeah, this is pretty impressive!

For this one, we looked at github search results and it seems like the majority of js code uses lowercase. Also, Chrome devtools forces lowercase.

![image](https://cloud.githubusercontent.com/assets/197597/23564900/6decf47c-0000-11e7-92f8-e1ef5f7391e0.png)